dev/benchmarks/macrobenchmarks/lib/common.dart

```diff
@@ -12,3 +12,4 @@ const String kLargeImagesRouteName = '/large_images';
 const String kTextRouteName = '/text';
 const String kAnimatedPlaceholderRouteName = '/animated_placeholder';
 const String kColorFilterAndFadeRouteName = '/color_filter_and_fade';
+const String kFadingChildAnimationRouteName = '/fading_child_animation';
```

Comment:
It seems that `fading_child_animation` doesn't match the page title of "Filtered Child Animation".

Reply:
The same page can apply any number of filters, but when it is invoked from this route it will do the opacity version. So this route is running the generic page with the opacity choice (via enum).

Is that a problem?

Reply:
I adapted the page to change its title depending on the variant it is running.